website/docs/docs/core/connect-data-platform/redshift-setup.md

```diff
@@ -92,10 +143,25 @@ Authentication, then your aws credentials are likely misconfigured. Try running
 `aws configure` to set up AWS access keys, and pick a default region. If you have any questions,
 please refer to the official AWS documentation on [Configuration and credential file settings](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
 
+
+| Profile field | Example | Description |
+| ------------- | ------- | ------------ |
+| `method` |IAM| use IAM to authenticate |
+| `iam_profile` | analyst | dbt will use the specified profile from your ~/.aws/config file |
```

Comment:
`host` is still required for IAM, right?
```suggestion
| `method` |IAM| use IAM to authenticate |
| `host` | hostname.region.redshift.amazonaws.com| Host of cluster |
| `iam_profile` | analyst | dbt will use the specified profile from your ~/.aws/config file |
```

Reply:
as per this [doc](https://github.com/aws/amazon-redshift-python-driver/blob/master/tutorials/001%20-%20Connecting%20to%20Amazon%20Redshift.ipynb), host is not required for IAM